rx-java/rxgrpc/src/main/resources/RxStub.mustache

```diff
@@ -121,6 +121,15 @@ public final class {{className}} {
                     {{/methods}}
                     .build();
         }
+        
+        public int getPrefetch(int methodId) {
+            return 512;
+        }
+        
+        public int getLowTide(int methodId) {
+            return 512 * 2 / 3;
+        }
```

Comment:
Why do these methods accept an unused `methodId`? Is there a reason a server implementor would want to set the prefetch and lowTide for a server, rather than using the hard coded values?

Reply:
We have a service with many methods. Some of these are consuming big objects, but others small ones. If service does not care it simply does not override. If ones care you can do it different for each method using id like:
```
    @Override
    public int getPrefetch(int methodId) {
        if (methodId == RxNassGrpc.METHODID_MULTIPLY) {
            return 8;
        }
        // Take default
        return super.getPrefetch(methodId);
    }
```

I just noticed that method ID is private constant :) We should modify generator template as well if you like this approach.

Reply:
We can make the methodids protected, since they would only be used in a subclass. Is there a way to make the API for setting server prefetch and lowtide more symmetric with the client? 

Client CallOptions can be set for the whole client stub (like your example above), or for an individual operation call at the call site.

Reply:
I have rewritten to use the same way as client:
```
    @Override
    protected CallOptions getCallOptions(int methodId) {
        return CallOptions.DEFAULT
            .withOption(ClientCalls.CALL_OPTIONS_PREFETCH, 8)
            .withOption(ClientCalls.CALL_OPTIONS_LOW_TIDE, 4);
    }
```